tests/test_user_manager.py

```diff
@@ -0,0 +1,55 @@
+"""Korone is a simple multipurpose Telegram Bot.
+This is a complete rewrite of PyKorone, refer to
+https://github.com/AmanoTeam/PyKorone and README.md for more
+information."""
+
+import os, sys
+
+sys.path.insert(0, f"{os.getcwd()}/src")   # makes possible to call
+                                           # from "korone" import ...
+
+from faker import Faker
+from pyrogram.types import User
+
+from src.korone.database import Database
+from src.korone.database.manager import UserManager, Cell, Column
+
+from pytest import fixture
+
+Database.connect("korone-test.db")
+Database.setup()
+
+
+class TestUserManager:
+    """Tests the management of user in database"""
+    user_manager = UserManager(Database)
```

Comment:
Perhaps you meant `UserManager(Database())` to pass in an instance of the Database to the UserManager?